adapter/ipListChecker/ipListChecker_test.go

```diff
@@ -109,14 +107,33 @@ func TestBasic(t *testing.T) {
 	lc := a.(*listChecker)
 
 	// do a NOP refresh of the same data
-	lc.refreshList()
+	if err := lc.fetchList(); err != nil {
+		t.Errorf("Unable to refresh list: %v", err)
+	}
 
-	// now try to parse the bad list
-	useGoodList = false
-	lc.refreshList()
-	list := lc.getList()
-	if len(list.payload) != 2 {
-		t.Errorf("Expecting %d, got %d entries", len(badList.WhiteList)-1, len(list.payload))
+	// now try to parse a list with errors
+	listToUse = 1
+	if err := lc.fetchList(); err != nil {
+		t.Errorf("Unable to refresh list: %v", err)
+	}
+	list := lc.getListState()
+	if len(list.entries) != 2 {
+		t.Errorf("Expecting %d, got %d entries", len(list1.WhiteList)-1, len(list.entries))
```

Comment:
Consider this from test code:

	if err != nil {
		t.Errorf("Unable to create aspect: %v", err)
	}

Based on the recommendation, this should be written as:

	if nil != err {
		t.Errorf("Creating an aspect failed with %v, expecting success", err)
	}

The if statement doesn't appear consistent with go conventions.